### PR TITLE
chore: add readme check in CI

### PR DESCRIPTION
**Pull request by bharathkkb**
fixes https://github.com/GoogleCloudPlatform/blueprints/issues/142

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,6 +45,7 @@ docker_run:
 		-e BILLING_ACCOUNT \
 		-e PROJECT_ID \
 		-e CLUSTER_NAME \
+        -e DOCKER_HOST_PATH=$(CURDIR) \
```
Comment (morgante) [inline Makefile:48]: Spacing looks inconsistent?

**Review by morgante — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (bharathkkb) [reply Makefile]: done

**Pull request merged**